src/css/_Styles.scss

```diff
@@ -1,8 +1,41 @@
 @import "_Variables";
 @import "_Mixins";
+@import "_LogIn";
 
-.hidden{
-  display: none;
+* {
+  box-sizing: border-box;
+  //border-color: red;
+}
+
+////*{
+////  border-color: red;
+//}
+
+.logo-area {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: auto;
+}
+
+h1 {
+  @include font-style-size($headline-font, 2.7em);
```

Comment:
Loving the use of mixins here & great work putting them in a separate partial file